docs/includes/generated_docs/schemas/beta.core.md

```diff
@@ -0,0 +1,357 @@
+# <strong>beta.core</strong> schema
+
+Available on backends: [**TPP**](../../backends#tpp), [**EMIS**](../../backends#emis)
+
+This schema defines the core tables and columns which should be available in any backend
+providing primary care data, allowing dataset definitions written using this schema to
+run across multiple backends.
+
+!!! warning
+    This schema is very much still a work-in-progress while the EMIS backed remains
+    under developemnt.
+
+``` {.python .copy title='To use this schema in an ehrQL file:'}
```

Comment:
thought: In the main documentation, we have `content.code.copy` enabled in `mkdocs.yml`. We don't in this repository. Maybe we should? (I don't think having the attribute matters here particularly.)

Reply:
That sounds sensible, but happy to leave that call to you.